unit/memory-analyzer/gdb_api.cpp

```diff
@@ -99,99 +94,117 @@ void gdb_api_internals_test()
   }
 }
 
+bool check_for_gdb()
+{
+  const bool has_gdb = run("gdb", {"gdb", "--version"}) == 0;
+
+  SECTION("check gdb is on the PATH")
+  {
+    REQUIRE(has_gdb);
+  }
+  return false;
```

Comment:
Doesn't this now always return `false`, and thus effectively disable all the tests?

Reply:
@tautschnig sorry about this type: should be fixed now.